lib/interface.js

```diff
@@ -146,6 +148,35 @@ Interface.prototype.getUser = function(endpointArn, callback) {
 
 
 /**
+ * Set a user's attributes by their EndpointArn
+ * @param {String}    endpointArn
+ * @param {Object}    attributes  An object that can contain string properties for CustomUserData, Enabled and Token
+ * @param {Function}  callback
+ */
+
+Interface.prototype.setAttributes = function(endpointArn, attributes, callback) {
+  var self = this;
+  if (typeof attributes == 'object') {
+    if (typeof attributes.CustomUserData == 'object') {
+      attributes.CustomUserData = JSON.stringify(attributes.CustomUserData);
```

Comment:
Of course - I'd be happy to :-) will be able to push the update later this morning.